### PR TITLE
Term entry c sharp sinh

### LINKED ISSUE #1764: [Term Entry] C# Math Functions `.Sinh()`
### Reason/inspiration (optional)

We would like a new term entry for [C#](https://www.codecademy.com/resources/docs/c-sharp) about the `Math.Sinh()` method.

The entry should go in a new file in `docs/content/c-sharp/concepts/math-functions/terms/sinh/sinh.md`

The entry should

- Have a syntax section illustrating the method with a ` ```pseudo ` code block.
- An example showing the method in use with a ` ```cs ` code block.
- A runnable codebyte example in a ` ```codebyte/csharp ` block.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by cslylla**
### Description

- Added new term for C# Math Functions: Math.Sinh()
- Closes #1764 

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/c-sharp/concepts/math-functions/terms/sinh/sinh.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+Math.Sinh(angle);
+```
+
+The `Math.Sinh()` method takes only one parameter, `angle`, an angle in radians of type `double`. The method returns the hyperbolic sine of the `angle` as a `double` value, or a `double` value equals to `angle` if the value of `angle` is:
+
+- `NaN` (not a number)
+- `NegativeInfinity`
+- `PositiveInfinity`
+
+> **Note:** Depending on the used operating system or architecture, the exact result or the input range of the `Math.Sinh()` method may differ due to the differences in the C runtime environment.
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/math-functions/terms/sinh/sinh.md:31]:
```suggestion
> **Note:** Depending on the operating system or architecture, the exact result or the input range of the `Math.Sinh()` method may differ due to the differences in the C runtime environment.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@cslylla I did a first review and only had one comment.

**Comment by cslylla** [comment]
@SSwiniarski I accepted the changes 👍🏻

**Review by SSwiniarski — APPROVED** [review]
@cslylla Looks good for a second review.

File: content/c-sharp/concepts/math-functions/terms/sinh/sinh.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+The following example first converts `88` degrees to radians, then uses the `Math.Sinh()` method to return the hyperbolic sine of that angle. `Math.Round()` rounds up the result to five decimals. Finally, the `Console.WriteLine()` function prints the result to the console:
+
+```cs
+using System;
+
+public class Example {
+  public static void Main(string[] args) {
+    double degrees = 88;
+    double radians = degrees * Math.PI/180;
+
```
Comment (yangc95) [inline content/c-sharp/concepts/math-functions/terms/sinh/sinh.md:44]:
```suggestion
```

**Comment by yangc95** [comment]
👏 @cslylla here's the live link: https://www.codecademy.com/resources/docs/c-sharp/math-functions/sinh

Screenshot 2023-03-28 at 2 22 29 PM

cc: @SSwiniarski

**Pull request merged**